tests/tom-swifty-test/SwiftRuntimeLibraryTests/SwiftDateTests.cs

```diff
@@ -17,6 +17,8 @@ namespace SwiftRuntimeLibraryTests {
 	[TestFixture]
 	public class SwiftDateTests {
 
+#if _MAC_TS_TEST_
```

Comment:
When is this set?

Reply:
I'm sorry I don't think I understand your question. I added this because to my understanding, this is a macOS specific test that was trying to be run on iOS device (VM) so it was causing errors. I used the same '#if _MAC_TS_TEST_' that some tests in the 'binding-tools-for-swift/tests/tom-swifty-test/SwiftReflector' use to specify a test for mac. Is there a different way I should do this?

Reply:
IIRC, `MAC_TS_TEST` only gets used inside code that was **generated** by a test.

Reply:
Actually let me double check

Reply:
Isn't it already being pinned to `macOS` with `platform: platform`?

Reply:
So without the `#if MAC_TS_TEST_`, binding-tools-for-swift/tests/tom-swifty-test/SwiftRuntimeLibraryTests/SwiftDateTests.cs was segfaulting as seen here: https://dev.azure.com/xamarin/public/_build/results?buildId=28617&view=logs&j=b944efd9-1a3a-54bf-926f-a01c570708bc&t=a206e760-4e0d-53e1-fe50-36a17b4c1b3f&l=22164 so it seemed to me that it was trying to run this test on iOS? @stephen-hawley

Reply:
Right, but it's never being run on the device. That looks like it's failing a leak test. What you did was comment the test out entirely. I think that you probably want to ignore this test and open an issue to look at it later.

Reply:
Okay sounds good!

Reply:
Removed the changes to this test!